javascript-if/main.js

```diff
@@ -0,0 +1,104 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+var shia = {
+  name: 'Shia Lebouf',
+  age: 34
+};
+
+
+function isOldEnoughToDrink(age) {
+
+  if (age => 21) {
+    return true;
+  } else {
+    return
+    (false)
+  };
+
+};
+
+
+function isOldEnoughToDrive(age) {
+  if (age => 18) {
+    return true
+    } else {
+      return false;
+    }
+  };
+
+
+  function isOldEnoughToDrinkAndDrive(age) {
+    return false;
+  };
+
+  function isUnderFive(number) {
+    if (number <= 5) {
+      return true;
+    } else {
+      return false
+    }
+  };
+
+  function isEven(number) {
+    if (number % 2 === 0) {
+      return true
+    } else {
+      return false
+    }
+  };
+  function startsWithJ(string) {
+    if(string[0] === 'J') {
+      return true
+    } else {
+        return false
+      }
+    };
```

Comment:
Looks like your indentation is a bit funky here too, please get this indentation to be consistent and follow proper code formatting rules.